src/hit/api/evaluator.h

```diff
@@ -345,6 +345,10 @@ namespace hit {
          */
         void relinearize_inplace(CKKSCiphertext &ct);
 
+        /* TODO
+         */
+        CKKSCiphertext bootstrap(const CKKSCiphertext &ct, bool rescale_for_bootstrapping = true);
```

Comment:
Nit: may add some boostrap implementation reference in future?

Reply:
Added docs.